mod_sample/media_info_parser.py

```diff
@@ -214,11 +217,13 @@ def generate_media_xml(sample) -> MediaInfoFetcher:
         if os.path.isfile(media_info_path):
             # Load media info, and replace full pathname
             tree = etree.parse(media_info_path)
-            for elem in tree.xpath("//track[@type='General']"):
-                for child in elem:
-                    if child.tag == 'Complete_name':
-                        child.text = child.text.replace(media_folder, '')
-                        break
+            root = tree.getroot()
+            try:
+                for rootElems in root:
+                    if rootElems.tag.split('}')[1] == 'media':
```

Comment:
There is no such tag as `Complete_name` in the XML generated by the latest version of mediainfo, but there is a tag `"media"` which has an attribute `ref` containing full path name of `media_folder`. When looped through the tags , every tag is prefixed with `'{https://mediaarea.net/mediainfo}'`, say for example the tag `media` when printed will look something like this `'{https://mediaarea.net/mediainfo}media'` , the split function is used to split the prefix and tag.

Reply:
I wonder if the prefixes can gotten rid of by loading the XSD in some way, but that's not that important to fix.

Reply:
Yes, [localname](https://stackoverflow.com/questions/18159221/remove-namespace-and-prefix-from-xml-in-python-using-lxml) can be used to eliminate those prefixes.

Reply:
That'd look a bit neater I'd say.

Don't forget about the unittests, they're still failing.

Reply:
Yes I'll update them :)